README.md

```diff
@@ -1,2 +1,21 @@
-# personal-portfolio
-Portfolio Website Project
+# Personal Portfolio Website
+Set up and Mobile Version Skeleton.
+
+# Built With
+- HTML
+- CSS
+- GitHub
+- Linters
+
+# To Run
+To run this project, you need to have a web browser such as Google Chrome, Safari, etc. And you need to clone the repository or download the ZIP files.
+
```

Comment:
- [ ] [OPTIONAL] Great job so far 👏 In addition to what you have done, I would suggest you add the `screenshot` of your landing page here in your README so as to give visitors a quick preview of what your portfolio looks like which can increase their interest in knowing you 😉 😎 
- [ ] [OPTIONAL] please elaborate more in the `Getting started` section. This is where you describe procedures to clone and run your project locally. It is better if you add some commands too 😉. This is the most important section for your fellow developers 😃 👍